### PR TITLE
[Feat]: card component added

### PR DESCRIPTION
**Pull request by aryan-13**
Added:
- Blog card
- Card with overlay
- Horizontal Card
- Vertical Card
- Card with Badge
- Card with Dismiss
- Card with only text 
cardss

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+/* ECOM CARD */
+.vertical-card {
+	margin: 1rem;
+	display: flex;
+	position: relative;
+	flex-direction: column;
+	width: 25rem;
+	padding: 1rem;
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
```
Comment (junaidshaikh-js) [inline components/card.css]: We don't need to add a vendor prefix unless you are trying to support a very old version browser.

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	padding: 1rem;
+	width: 36rem;
+	position: relative;
+	background-color: var(--color-white);
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	-moz-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+}
+.img-box,
+img {
```
Comment (junaidshaikh-js) [inline components/card.css]: Repeated styles can be combined lines 14, 58

Also, this can be moved to reset.css

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+	left: 6%;
+}
+.heading-top {
+	order: -1;
+}
+.flex {
+	display: flex;
+}
+.small-img {
+	/* height: 110px; */
```
Comment (junaidshaikh-js) [inline components/card.css]: code comment should be removed.

**Review by junaidshaikh-js — COMMENTED** [review]
You can reduce the indentation level of your code formatter to avoid large space in the code, you can keep indentation of two spaces.

File: components/card.css [hunk 4]
```diff
@@ -24,11 +29,81 @@
...
+	color: var(--color-white);
+	padding: 7px 1.3rem;
+}
+/* CARD DISMISS BTN */
+.card-dismiss {
+	position: absolute;
+	color: var(--color-white);
+	padding: 7px 1.3rem;
+	padding: 7px 1.3rem;
+	right: 0rem;
```
Comment (hinanshisuthar) [inline components/card.css]: not necessary to mention unit when the value is 0

File: components/card.css (hunk 2 above)
Comment (aryan-13) [reply components/card.css]: Okay

File: components/card.css (hunk 3 above)
Comment (aryan-13) [reply components/card.css]: Done

File: components/card.css [hunk 5]
```diff
@@ -24,11 +29,81 @@
...
+img {
+	width: 100%;
+}
+
+.card-heading-primary {
+	margin: 0;
+}
+.card-heading-sub {
+	color: var(--color-grey-dark);
+	margin: 0;
```
Comment (hinanshisuthar) [inline components/card.css:60]: in line 63 and 67 the css rule declarations are same. Hence, advised to use a variable instead.

**Review by hinanshisuthar — CHANGES_REQUESTED** [review]
This much indentation is not required. Would suggest to beautify the file.

File: components/card.css (hunk 1 above)
Comment (aryan-13) [reply components/card.css]: Okayy

File: components/card.css (hunk 4 above)
Comment (aryan-13) [reply components/card.css]: Removed it

**Pull request merged**